src/app/vault/vault.component.ts

```diff
@@ -102,9 +102,18 @@ export class VaultComponent implements OnInit, OnDestroy {
                     case 'newFolder':
                         await this.addFolder();
                         break;
+                    case 'copyUsername':
+                        if (this.selectedCipher() != null && this.selectedCipher().login.username != null) {
+                            this.copyValue(this.selectedCipher().login.username, 'username');
+                        }
+                        break;
+                    case 'copyPassword':
+                        console.log(this.selectedCipher());
```

Comment:
The console.log() is probably not supposed to stay or is it?

Reply:
Yes, I forgot to push this change.